src/plugins/misc/commands/jumbo.command.ts

```diff
@@ -0,0 +1,52 @@
+import {ChatInputCommandInteraction, inlineCode, PermissionFlagsBits} from "discord.js";
+import {SlashCommandBuilder, SlashCommandScope} from "../../../builders/SlashCommandBuilder";
+import {useChatCommand} from "../../../hooks/useChatCommand";
+import axios from 'axios';
+
+const builder = new SlashCommandBuilder()
+    .setName("jumbo")
+    .setDescription("Make G'day blow up an image.")
+    .addStringOption(option => option.setName("emoji").setDescription("Emoji to jumbo size.").setRequired(true))
+    .setDefaultMemberPermissions(PermissionFlagsBits.ManageMessages)
+    .setEphemeral(true)
```

Comment:
This command shouldn't be ephemeral

Reply:
This will likely be referenced later but wanted to make this mention:
I kept emphemeral due to the need of a message, since I was still learning how useChatCommand works. I'm starting to learn more about it and I was able to realize I can just return the emoji content instead of doing it separately.

Tl;dr, removed.